mempool/src/shared_mempool/types.rs

```diff
@@ -217,15 +218,16 @@ pub type MempoolEventsReceiver = mpsc::Receiver<MempoolClientRequest>;
 /// `is_alive` - is connection healthy
 #[derive(Clone, Debug)]
 pub(crate) struct PeerSyncState {
-    pub timeline_id: u64,
+    pub timeline_id: TimelineId,
     pub broadcast_info: BroadcastInfo,
     pub metadata: ConnectionMetadata,
 }
 
+// TODO: pass in number of buckets to initialize timeline_id instead of None?
```

Comment:
this is resolved? or I am missing what this is referring to?

Reply:
Yup, stray comment :-/